requirements/base.in

```diff
@@ -4,7 +4,7 @@ psycopg2-binary~=2.8
 pytz==2020.1
 whitenoise~=4.1
 django-crispy-forms~=1.8
-Django~=2.2
+Django~=3.0
```

Comment:
This feels like a major change to mix into this! Should it be a separate PR?

Reply:
As you commented on #138, Django 3 is a surprisingly small update. I am content to have this in here, albeit in a distinct commit, for that reason.